app/components/global-navigation/search.js

```diff
@@ -355,6 +355,9 @@ export default Component.extend({
 				this.inputField.blur();
 
 				this.closeSearch();
+			} else if (this.goToSearchResults) {
```

Comment:
it actually works for me on prod and preview. Is the bug on sandbox somewhere?

Reply:
The bug is in current dev branch, reproducible if you run it e.g. on devbox